### DISCUSSION TITLE
Cannot find .glb file when site loads

### DISCUSSION THREAD

**Discussion by itwela** [vercel/community#3889] [Help] [answered]
### Page to Investigate

https://itwelaportfolio.vercel.app/

### Steps to Reproduce

Just go on the homepage and pull up the inspector. It will give you an error that it cant find this file that im looking at in my vercel directory. It sucks because everything else loaded on the site except for that one file and i cant figure out why....

**Comment by LucasGuzman3D** [comment]
I see that the 3D model is live now... how did you fix it? I'm having the same issue. 404 on my glb file.

**Reply by itwela** [reply, marked as answer]
I'm glad you asked! So I was not aware but I had to put any media i had for the project in the **public** folder. 

If you don't have a public folder(i didn't have one), make one, and put all images, 3d files, etc in them. (You have to make the public folder directly under the root directory. So if your root folders name is "site", it needs to be made like, "site/public"

But because of that, it left me with a problem though. 

I was referencing my images from the old location all through my code and didn't know how to fix that. 

I'm not sure how I had the file path before I changed it, but once I made the folder I referenced all my images like this ("image_name"). That's it. I didn't even have to put ("public/image_name") as the path.

Literally just referenced them like that once they were in the public folder and it fixed it.

**Comment by LucasGuzman3D** [comment]
Oh woah! That did it! Thank you so much for your reply!

**Reply by itwela** [reply]
no problem, glad i could help !

**Comment by shrxyyya** [comment]
I had re-deployed my project so many times but wasn't able to figure out the error. This reply of yours helped me so much thanks a lot!
Have a great day!
